### PR TITLE
feat(store blueprint): add a blueprint for customizing the store pass…

### PR DESCRIPTION
**Pull request by velrest**
…ed to emeis engine

### CODE REVIEW

File: blueprints/emeis-store/files/__root__/services/__name__.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import Store from "@ember-data/store";
+
+export default class <%= classifiedModuleName %>Store extends Store {
+  adapter = "<%= dasherizedModuleName %>";
+}
```
Comment (anehx) [inline blueprints/emeis-store/files/__root__/services/__name__.js:5]: I assume that `__name__` in the filename is the module name, right? If so, shouldn't the filename be something like `__name__-store` to match the classname?

Comment (velrest) [reply blueprints/emeis-store/files/__root__/services/__name__.js:5]: IIRC if you generate a component in a non-pod app, the file name ist `components/users.js` or `components/users/index.js`.

This blueprint has no pod support for now.

Comment (anehx) [reply blueprints/emeis-store/files/__root__/services/__name__.js:5]: Ah, I missunderstood the blueprint, NVM

**Pull request merged**